utils/types.ts

```diff
@@ -1,3 +1,5 @@
+import { string } from "square/dist/schema";
```

Comment:
This might've been auto-imported from somewhere, but it isn't used anywhere in the file. It should be safe to remove it.

Reply:
Sounds good, I'll go ahead and remove it before merging